graviti/dataframe/column/series.py

```diff
@@ -7,10 +7,16 @@
 
 
 from itertools import islice
-from typing import Any, Dict, Iterable, List, Optional, Sequence, Union, overload
+from typing import Any, Dict, Iterable, List, Optional, Sequence, Type, TypeVar, Union, overload
```

Comment:
in commit message `serires` -> `series`

you can just use `implement "column.Series.from_pyarrow"`